src/writer.rs

```diff
@@ -208,6 +212,30 @@ impl SVGWriter {
             w.start_element("path");
             w.write_attribute("d", &symbol.path);
             w.end_element();
+            if let Some(origin) = symbol.origin {
+                let x = origin.x();
+                let y = origin.y();
+                w.start_element("path");
+                let crosshair_size = 100. * self.transform.extract_scale().x();
+                let crosshair = format!(
+                    " M{},{} L{},{} M{},{} L{},{}",
+                    x - crosshair_size,
+                    y,
+                    x + crosshair_size,
+                    y,
+                    x,
+                    y - crosshair_size,
+                    x,
+                    y + crosshair_size
+                );
```

Comment:
Very minor: perhaps we can factor out these coordinates for clarity. Something like:

```rust
let xl = x - crosshair_size;
let xr = x + crosshair_size;
let yb = y - crosshair_size;
let yt = y + crosshair_size;
let crosshair = format!(" M{},{} L{},{} M{},{} L{},{}", xl, y, xr, y, x, yb, x, yt);
```

Though, it's fairly obvious what's going on here, so feel free to ignore this suggestion.

Reply:
I considered extracting that whole chunk of code into a function. That you commented on it suggests I should do that as well as consider this suggestion.